### PR TITLE
First story book commit with sample stories

### PR DESCRIPTION
**Pull request by ice-rahul**
screenshot-localhost-6006-1616015082827

### CODE REVIEW

File: src/components/Button/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import './button.css'
+
+export interface ButtonProps {
```
Comment (michaelspeed) [inline src/components/Button/index.tsx:4]: exporting this to a different file will be much better

Comment (michaelspeed) [reply src/components/Button/index.tsx:4]: something like ButtonProps.ts

File: src/components/Button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
```
Comment (michaelspeed) [inline src/components/Button/index.tsx:1]: The component should not be on the index file

**Review by michaelspeed — CHANGES_REQUESTED** [review]

File: src/components/Button/index.tsx (hunk 2 above)
Comment (ice-rahul) [reply src/components/Button/index.tsx:1]: If we will export it from Button.tsx
Then while importing this component we would have to do something like
import Button from 'src/component/Button/Button'

Where as exporting it from index.tsx will reduce the last Button path from import statement

***Should we*** create both Button.tsx and Index.tsx and export Button from Index.tsx

File: src/components/Button/index.tsx (hunk 1 above)
Comment (ice-rahul) [reply src/components/Button/index.tsx:4]: Ok

File: src/components/Button/index.tsx (hunk 2 above)
Comment (michaelspeed) [reply src/components/Button/index.tsx:1]: Yes its will be like a single folder exporting all the components and props through the index.ts file inside it. but will be much more maintainable

**Pull request merged**